ampd/src/monitoring/mod.rs

```diff
@@ -0,0 +1,69 @@
+use std::net::{Ipv4Addr, SocketAddrV4};
+
+use serde::{Deserialize, Serialize};
+
+mod endpoints;
+pub mod server;
+
+#[derive(Clone)]
+pub enum MetricsMsg {
+    IncBlockReceived,
+}
+
+#[derive(Serialize, Deserialize, Clone, PartialEq, Debug)]
+pub struct Config {
+    pub enabled: bool,
+    pub bind_address: SocketAddrV4,
```

Comment:
```suggestion
    enabled: bool,
    bind_address: SocketAddrV4,
```
we want the usage to go through the bind_addr() method